### PR TITLE
Feat/create mkdocs documentation

### PR DESCRIPTION
**Pull request by WilliGautier**

### CODE REVIEW

File: scripts/release.sh [hunk 1]
```diff
@@ -2,89 +2,15 @@
...
-    conf/restart_strategy.env \
-    conf/stack.env \
-    conf/arlas_iam.env \
-    conf/postgres.env \
-    > docker_compose_services_aias.md
-
-git add docker_compose_services_*.md
-git commit -m "update docker compose services documentation"
+# Tag the version
+git add docs/docs/dc_services/docker_compose_services_*.md
```
Comment (sylvaingaudan) [inline scripts/release.sh:13]: this is not the same as listing the files: some are intentionnaly missing.

File: scripts/release.sh [hunk 2]
```diff
@@ -2,89 +2,15 @@
...
-    conf/permissions.env \
-    conf/apisix.env \
-    conf/restart_strategy.env \
-    conf/stack.env \
-    conf/arlas_iam.env \
-    conf/postgres.env \
-    > docker_compose_services_iam.md
+# Publish documentation
+pip3.10 install mkdocs-material termynal
+mkdocs gh-deploy -f docs/mkdocs.yml
```
Comment (sylvaingaudan) [inline scripts/release.sh:10]: this is not the same as listing the files: some are intentionnaly missing.

File: scripts/release.sh [hunk 3]
```diff
@@ -2,89 +2,15 @@
...
-    conf/elastic.env \
-    conf/arlas.env \
-    conf/persistence-file.env \
-    conf/permissions.env \
-    conf/apisix.env \
-    conf/restart_strategy.env \
-    conf/stack.env \
-    > docker_compose_services_simple.md
+# Generate the md documentation
+./mkDocs.sh
```
Comment (sylvaingaudan) [inline scripts/release.sh:6]: this is not the same as listing the files: some are intentionnaly missing.

**Review by sylvaingaudan — CHANGES_REQUESTED** [review]

**Review by QuCMGisaia — APPROVED** [review]

**Review by MohamedHamouGisaia — COMMENTED** [review]
LGTM, a part from the generated docker-composes services markdown files, (@sylvaingaudan's comments)

File: scripts/release.sh (hunk 3 above)
Comment (WilliGautier) [reply scripts/release.sh:6]: I don't understand what you mean by 'listing the files".
I didn't change those lines, I just moved it into `./mkDocs.sh`

**Review by sylvaingaudan — APPROVED** [review]

**Pull request merged**